### PR TITLE
classification

### PR DESCRIPTION
**Pull request by jowonjin96**
classification model1

### CODE REVIEW

File: class02/homework/jowonjin/hw1/led.py
Comment (JerryJLee) [inline class02/homework/jowonjin/hw1/led.py:1]: 예전 과제에 쓰였던 파일은 해당 커밋에 포함되면 안되겠죠?

File: class02/homework/jowonjin/hw2/hw03_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+./splitted_dataset/val: 12
+./splitted_dataset/train/<class#>: 6
+./splitted_datase
+t/train/<class#>: 6
+```
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|
+|MobileNet-V3-large-1x|1.0 ||213.68|0:00:06.715144||64|0.0046799|-|
```
Comment (JerryJLee) [inline class02/homework/jowonjin/hw2/hw03_template.md:20]: 3가지 모델은 어디로??

Comment (jowonjin96) [reply class02/homework/jowonjin/hw2/hw03_template.md:20]: 1가지 이상 모델 사용해보라고 하셔서 1가지 모델 적용만 해보았습니다..
